data/timesketch.conf

```diff
@@ -220,3 +220,9 @@ EMAIL_RECIPIENTS = []
 
 # Configuration to construct URLs for resources.
 EXTERNAL_HOST_URL = 'https://localhost'
+
+#-------------------------------------------------------------------------------
+# Sigma Settings
+
+SIGMA_RULES_FOLDER = '../../../../data/sigma/rules/'
```

Comment:
@berggren what would be the best setting here that would work for Docker dev systems assuming for all others we would need to adjust the dev scripts?

Reply:
The rules are copied to /etc/timesketch/sigma/rules in the new docker deployment helper script. So we should assume that folder being the default for the bundled rules.

Followup question: Do we want another config param for a path to self hosted rules?

Reply:
What do you mean with "self hosted rules" Imho we will only have one folder, where people would deploy their rules. So it does not matter if the pull form Sigma upstream, have a copy of "internal rules" or use the one shipped with Timesketch, all will end in the same folder.

WRT config setting, can I also assume that the sigma config with the mappings etc will end in /etc/timesketch/sigma/ ?

Reply:
So the idea here is to have a key in the ```timesketch.conf```for 
```SIGMA_RULES_FOLDER = '../../../../data/sigma/rules/'
SIGMA_CONFIG = '../../../../data/sigma_config.yaml'
```

That work for docker dev, docker e2e and other, but I am not sure what values I need to set and where I should modify the deployment scripts if needed.

Reply:
ok discussed this morning directly and this should now be way better moving forward